### PR TITLE
AC Automated Test - LRAC-9102, LRAC-9105 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LRAC-9102
https://issues.liferay.com/browse/LRAC-9105

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase [hunk 1]
```diff
@@ -408,12 +415,19 @@ definition {
...
 	@priority = "3"
 	test PropertyCanHaveABigName {
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
 
-		// TODO LRAC-9102 PropertyCanHaveABigName pending implementation
+		ACSettings.goToProperties();
+
+		ACProperties.addNewProperty(propertyName = "This is a property with a big name to ckeck the limit of a name!!");
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:426]: hi @catarineaguiar , I would suggest you add one more character in order to validate the boundary of this field, seeing the "Exceeds maximum length." message, and then you remove the additional char and proceed with the creation.

What do you think?

cc: @ViniLopes20

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:426]: Additionally, we could validate that Save button is disabled ;)

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase:426]: hey guys, let's keep it as is. I just noticed a different test that is related to this boundaries validation https://issues.liferay.com/browse/LRAC-9106.

Thanks!

**Comment by alexcandidos** [comment]
Hi @catarineaguiar , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**